cmd/root.go

```diff
@@ -45,23 +40,25 @@ func init() {
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
 
-	// RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.tmc.yaml)")
+	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", config.DefaultConfigDir, "path to config directory")
 	RootCmd.PersistentFlags().StringVarP(&loglevel, "loglevel", "l", "", "enable logging by setting a log level, one of [error, warn, info, debug, off]")
 	RootCmd.PersistentPreRun = preRunAll
 	config.InitViper()
```

Comment:
Can you please check if we can remove this line ? 
I could imagine InitViper() is not needed anymore at this place, as initialization is now always done in preRunAll()

Reply:
We cannot. This first `InitViper()`is necessary to initialize the env var bindings at the right time.

Reply:
I think I've managed to split the initialization into clearly separated parts now. One for initializing primary viper settings and one for reading in the config file. Seems like nothing got broken.